Examples/KleinGordon/KleinGordonLevel.cpp

```diff
@@ -0,0 +1,159 @@
+#include "KleinGordonLevel.hpp"
+#include "FixedGridsTagger.hpp"
+#include "FourthOrderDerivatives.hpp"
+#include "KleinGordonRHS.hpp"
+#include <numeric>
+
+using namespace amrex;
+
+void KleinGordonLevel::variableSetUp()
+{
+    BL_PROFILE("KleinGordonLevel::variableSetUp()");
+
+    // Set up the state variables
+    stateVariableSetUp();
+
+    // Set up derived variables
+    derive_lst.add(
+        "analytic_soln", amrex::IndexType::TheCellType(), 1, calc_derive_mf,
+        [=](const amrex::Box &box)
+        { return amrex::grow(box, simParams().num_ghosts); },
+        &amrex::cell_quartic_interp);
+
+    derive_lst.addComponent("analytic_soln", desc_lst, State_Type, 0, 1);
+}
+
+void KleinGordonLevel::initData()
+{
+    BL_PROFILE("KleinGordonLevel::initData()");
+
+    const auto problo = geom.ProbLoArray();
+    const auto dx     = geom.CellSizeArray();
+
+    std::array<double, AMREX_SPACEDIM> center{};
+
+    amrex::ParmParse pp;
+    pp.query("center", center);
+
+    MultiFab &S_new  = get_new_data(State_Type);
```

Comment:
For consistency with the rest of the file (and our naming conventions).
```suggestion
    MultiFab &state_new  = get_new_data(State_Type);
```
Yes I know this is a problem with the BinaryBH example but we might as well get it right here!